diff --git a/fixtures/filter-omission/TextTools.slnf b/fixtures/filter-omission/TextTools.slnf
index ca12560..547dd76 100644
--- a/fixtures/filter-omission/TextTools.slnf
+++ b/fixtures/filter-omission/TextTools.slnf
@@ -2,7 +2,8 @@
   "solution": {
     "path": "TextTools.sln",
     "projects": [
-      "src\\TextTools\\TextTools.csproj"
+      "src\\TextTools\\TextTools.csproj",
+      "tests\\TextTools.Tests\\TextTools.Tests.csproj"
     ]
   }
 }
